### ISSUE TITLE
Allow slashes in hparam / metric names

### ISSUE THREAD

**Issue by its-dron** [aimhubio/aim#580]
Aim throws an error when attempting to log a value with a name that contains a `/`. Using forward slashes is common practice for metric grouping in TensorBoard and other popular experiment tracking systems. Almost every code sample you'll find logs `train/loss` and `val/loss`, as an example.

If there is a good reason for this restriction, then there should be more clear errors and/or documentation to cover it.

Example error thrown when calling `aim.pytorch_lightning.AimLogger.log_hyperparameters`:

```
Something went wrong in `_track_body`. The process will continue to execute.
`_track_body`: dictionary item key must be a python identifier: `model/params_total`
```

Version info:

```
aim                               2.7.1
python                            3.8.0
pytorch-lightning                 1.4.2
torch                             1.9.0
```

**Comment by gorarakelyan** [comment]
Hi @its-dron, thanks for the feedback.
The restriction was imposed in order to keep Aim query language lucid and clean. However, based on the feedback, it was removed and in the upcoming release any metric/param name will be allowed.

**Comment by gorarakelyan** [comment]
Restriction was removed since Aim v3
